docs/application/dotnet/getting-started/sec-privileges.md

```diff
@@ -0,0 +1,189 @@
+
+# Security and API Privileges
+
+To effectively protect the device system and user private data, the
+Tizen security architecture is based on privileges and application
+signing of the Linux basic security model, which includes process
+isolation and mandatory access control. Since Tizen, as an open mobile
```

Comment:
In line 7-8, "Since Tizen, as an open mobile platform" 

I think "mobile" should be removed.

Reply:
removed